model/src/test/kotlin/AdvisorResultContainerTest.kt

```diff
@@ -32,10 +32,22 @@ import java.time.Instant
 class AdvisorResultContainerTest : WordSpec() {
     private val id = Identifier("type", "namespace", "name", "version")
 
-    private val vulnerability11 = Vulnerability("CVE-11", 1.1F, URI("https://1.1.com"))
-    private val vulnerability12 = Vulnerability("CVE-12", 1.2F, URI("https://1.2.com"))
-    private val vulnerability21 = Vulnerability("CVE-21", 2.1F)
-    private val vulnerability22 = Vulnerability("CVE-22", 2.2F)
+    private val vulnerability11 = Vulnerability(
```

Comment:
The changes in this test should be in the next commit since only then you have a `VulnerabilityReference` in the `Vulnerability` data model.